File: C++/SubtractProductAndSum.cpp
//#include <iostream>
//#include <vector>
//#include <string>
//using namespace std;
//
//int subtractProductAndSum(int n) {
//	string str = to_string(n);
//	vector<int> list;
//	int product = 1;
//	int sum = 0;
//	for (int i = 0; i < str.size(); i++) {
//		string s(1, str[i]);
//		list.push_back(stoi(s));
//	}
//	for (int j = 0; j < list.size(); j++) {
//		product = product * list[j];
//
//	}
//	for (int k = 0; k < list.size(); k++) {
//		sum = sum + list[k];
//
//	}
//	return product - sum;
//}
//
//
//int main() {
//	int n = 4421;
//	int out;
//	out = subtractProductAndSum(n);
//	cout << out << endl;
//	return 0;
//}

File: Python/SubtractProductandSum.py
def subtractProductAndSum(n):
    x = str(n)
    lst = []
    product = 1
    sum = 0
    for i in x:
        lst.append(int(i))
    for j in lst:
        product = product * j
    for k in lst:
        sum = sum + k
    out = product - sum
    return out


n = 234
out = subtractProductAndSum(n)
print(out)
